frontends/web/src/components/headerssync/headerssync.tsx

```diff
@@ -1,5 +1,7 @@
+
 /**
- * Copyright 2018 Shift Devices AG
+ * Copyright 2018  Shift Devices AG
+ * Copyright 2020 Shift Crypto AG
```

Comment:
2021?

Reply:
well at the time it was right :)

![Screen Shot 2021-02-10 at 4 11 44 PM](https://user-images.githubusercontent.com/546900/107528966-b209ae00-6bba-11eb-9263-9158cc4f58b7.png)